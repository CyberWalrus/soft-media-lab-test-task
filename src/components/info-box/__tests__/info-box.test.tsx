import React from 'react';
import { shallow } from 'enzyme';
import InfoBox from '../info-box';

describe('InfoBox component', () => {
  it('should renders correctly', () => {
    const wrapper = shallow(<InfoBox salary={0} ndfl />);
    expect(wrapper.isEmptyRender()).not.toEqual(true);

    expect(wrapper.html()).toMatchSnapshot();
  });
});
