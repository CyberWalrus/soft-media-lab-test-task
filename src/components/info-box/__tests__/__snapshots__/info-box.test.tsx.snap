// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`InfoBox component should renders correctly 1`] = `"<div class=\\"info-box\\"><div><b class=\\"info-box__amount\\">0</b><span class=\\"info-box__text\\">₽ сотрудник будит получать на руки</span></div><div><b class=\\"info-box__amount\\">0</b><span class=\\"info-box__text\\">₽ НДФЛ, 13% от оклада</span></div><div><b class=\\"info-box__amount\\">0</b><span class=\\"info-box__text\\">₽ за сотрудника в месяц</span></div></div>"`;
